authority/voting/server/wire_handler.go

```diff
@@ -149,7 +149,6 @@ func (s *Server) onGetConsensus(rAddr net.Addr, cmd *commands.GetConsensus) comm
 	resp := &commands.Consensus{}
 	doc, err := s.state.documentForEpoch(cmd.Epoch)
 	if err != nil {
-		s.log.Errorf("Peer %v: Failed to retrieve document for epoch '%v': %v", rAddr, cmd.Epoch, err)
```

Comment:
Are you sure you want to completely remove the entire log line instead of just modifying it to omit the ip address?

Reply:
yes